libs/ledger-live-common/dependencies.md

```diff
@@ -1,4 +1,4 @@
-This document allows to track, explain and maintain the dependencies we have defined in package.json.
+This document allows to track, ex.json.plain and maintain the dependencies we have defined in package
```

Comment:
There seems to be a stray `.json.` here?

Reply:
Done!